cpprevolve/revolve/brains/controller/DifferentialCPG.h

```diff
@@ -33,25 +37,25 @@ class DifferentialCPG
     };
 
     /// \brief Constructor
-    /// \param[in] _modelName Name of the robot
-    /// \param[in] _node The brain node
-    /// \param[in] _motors Reference to a motor list, it be reordered
-    /// \param[in] _sensors Reference to a sensor list, it might be reordered
+    /// \param[in] params Parameters for the controller
+    /// \param[in] _actuators Reference to a actuator list
+    /// \param[in] config_cppn_genome Reference to the genome for configuring the weights in CPG
     DifferentialCPG(
-            const ControllerParams &params,
-            const std::vector< std::unique_ptr < Actuator > > &_actuators);
+            const DifferentialCPG::ControllerParams &params,
+            const std::vector<std::shared_ptr<Actuator>> &_actuators,
+            NEAT::Genome config_cppn_genome);
```

Comment:
What if I want to load a DifferentialCPG without a cppn, but just normally with the weights. That's why the CPPN implementation should be in a subclass. Like `CPPNDifferentialCPG`